package.json

```diff
@@ -123,7 +123,7 @@
     "stylelint-config-recommended": "^9.0.0",
     "stylelint-selector-bem-pattern": "^2.1.1",
     "svg-to-ts": "^8.9.0",
-    "svgo": "^2.8.0",
+    "svgo": "^3.0.2",
```

Comment:
Will need to check this breaking change.

Reply:
Fixed: https://github.com/Legal-and-General/canopy/pull/1100/commits/4eca6bb4514aba6fe23e71b7622dda5d9b6f93f9